pages/Documents/MessagingChannels/PushNotificationService/Configuration/push-proxy.md

```diff
@@ -14,6 +14,8 @@ permalink: push-notification-service-configuration-push-proxy.html
 indicator: messaging
 ---
 
+![](img/pusher/PushProxyFlow.png)
```

Comment:
Please add a text alternative (“alt text”). (If possible, for other images on the page, too.)